src/status_im/contexts/wallet/create_account/select_keypair/view.cljs

```diff
@@ -74,7 +74,6 @@
         selected-keypair                        (rf/sub [:wallet/selected-keypair-uid])
         profile-picture                         (rf/sub [:profile/image])
         [selected-key-uid set-selected-key-uid] (rn/use-state selected-keypair)]
-    (rn/use-mount #(rf/dispatch [:wallet/get-keypairs]))
```

Comment:
Removed the event here as the Keypair details are fetched whenever a new account is added or an account is removed. This helps in resolving this issue and also displaying the newly generated keypairs in the wallet settings later (without having to re-login).

Reply:
@smohamedjavid after you create a new keypair does it still show in the list of keypairs?

Reply:
Yes, @OmarBasem as we request for keypairs after account creation.

https://github.com/status-im/status-mobile/pull/19505/files#diff-694ffc2f895558df8b19c6c44a82bfdc8e8e7d8a9743243a0b66958fb39740e4R184